### ISSUE TITLE
Announcement: Judge0 API goes Freemium

### ISSUE THREAD

**Issue by hermanzdosilovic** [judge0/api#171]
**tl;dr** Since [**June 1st, 2020**](https://www.timeanddate.com/countdown/generic?iso=20200601T000001&p0=281&msg=Judge0+API+goes+Freemium&font=sanserif) https://api.judge0.com (standard edition of the Judge0 API) will only be accessible through [RapidAPI](https://rapidapi.com/hermanzdosilovic/api/judge0).

**tl;dr** I am creating new email newsletter subscription list. If you want to receive at most one email per month with the latest Judge0 related news and updates, please fill [this form](https://forms.gle/p3KpGYSjxAbvGUft9) (30 seconds of your time).

![4AduA0Q](https://user-images.githubusercontent.com/7927278/81705227-19010780-946f-11ea-8fcd-e8d773847600.png)

### Introduction
I have started working on the Judge0 API on August 2016 with the vision of creating free, robust and open-source web API for code execution, and without [isolate](https://github.com/ioi/isolate), [Docker](https://github.com/docker), [Ruby on Rails](https://github.com/rails/rails), and many other open-source projects this would not be possible. This vision and passion still exists, but to make Judge0 even better I need to spend more time into it. That is why after almost four years I have decided to go Freemium with the Judge0 API.

Freemium means that Judge0 API will still be open-source and freely self-hosted, but if you would like to use public official instance (https://api.judge0.com), maintained by me, you would need to pay for some features while basic functionality will still be free.

### Achievments and Progress
In these almost four years Judge0 API has:
- over 7.3 million submissions (code executions) on the public instance,
- support for [42 languges](https://preview.api.judge0.com/languages) in standard edition and [10 languages](https://preview.extra.api.judge0.com/languages) in extra edition,
- [31 releases](https://github.com/judge0/api/releases),
- [325 stargazers](https://github.com/judge0/api/stargazers),
- [scientific paper](https://minio.judge0.com/public/paper-in-review.pdf),
- more than [46](https://github.com/judge0/api#references) projects, companies and organizations that actively use it, and
- [136 resolved issues](https://github.com/judge0/api/issues?q=is%3Aissue+is%3Aclosed).

Moreover, [Judge0 IDE](https://ide.judge0.com) has more than 1K active users per month, and @skilldeliver has joined the Judge0 and made [Judge0 Discord Bot](https://github.com/judge0/discord-bot).

Furthermore, I have spend around 2k hours on Judge0 related work and answered on more than 200 emails.

### What stays the same
- All Judge0 projects stay open-source.
- Judge0 API stays freely self-hosted with all the features.
- Judge0 IDE stays free and open-source.

### What changes
- Public instance of the Judge0 API will be accessible through [RapidAPI](https://rapidapi.com/hermanzdosilovic/api/judge0).

### What should you do
If you currently use https://api.judge0.com you should:
1. Sign up on the [RapidAPI](https://rapidapi.com).
2. [Choose a plan](https://rapidapi.com/hermanzdosilovic/api/judge0) for the Judge0 API.
3. Use the API endpoint provided by the RapidAPI with the API token they give you.

or if provided plans don't work for you, you can always [self-host](https://github.com/judge0/api#installation).

---
I am creating new email newsletter subscription list. If you want to receive at most one email per month with the latest Judge0 related news and updates, please fill [this form](https://forms.gle/p3KpGYSjxAbvGUft9) (30 seconds of your time).

Let me know if you have any questions or feedback in the comments below.

Best regards,
Herman
